CRM/Dedupe/Merger.php

```diff
@@ -1092,8 +1092,7 @@ public static function getRowsElementsAndInfo($mainId, $otherId, $checkPermissio
         // CRM-15681 don't display sub-types in UI
         continue;
       }
-      foreach (['main', 'other'] as $moniker) {
-        $contact = &$$moniker;
+      foreach (['main' => $main, 'other' => $other] as $moniker => $contact) {
```

Comment:
It looks like in the "before" version the variable was being assigned by reference but now the ampersand has been removed. Is that an issue?

Reply:
It's not actually altered so it should be fine